README.rst

```diff
@@ -2,17 +2,72 @@
 taclib
 ======
 
+Taclib stands for **Task as Containers Library**, and it was created to make easy
+to run luigi tasks in Kubernetes.
 
 
+Cluster Role
+------------
+Each luigi task will be running in a pod as a containerized application. The K8s pods
+have to have permission from the cluster to create new pods, in order to that we have
```

Comment:
Not entirely correct, only the controller (luigi worker) pod needs to have this permissions, the tasks themselves don't spawn other tasks

Reply:
...in order to *do* that